### PR TITLE
ci(yarn/node): use fixed versions and upgrade

### PR DESCRIPTION
**Pull request by vvo**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -7,26 +7,24 @@ jobs:
...
       # Download and cache dependencies
       - restore_cache:
           keys:
           - v1-dependencies-{{ checksum "package.json" }}
           # fallback to using the latest cache if no exact match is found
           - v1-dependencies-
 
-      - run: yarn install
+      - run: yarn
+      - run: npm rebuild node-sass --force
```
Comment (samouss) [inline .circleci/config.yml]: Why do we need to rebuild `node-sass`? Maybe add a comment.

File: community-website/src/community-project-boilerplate-docgen/package.json [hunk 2]
```diff
@@ -3,52 +3,52 @@
...
     "sass-loader": "6.0.6",
     "stat-mode": "0.2.2",
-    "style-loader": "0.19.0",
-    "uglifyjs-webpack-plugin": "1.1.2",
+    "style-loader": "0.21.0",
+    "uglifyjs-webpack-plugin": "1.2.6",
     "webpack": "3.10.0",
     "webpack-dev-middleware": "2.0.4",
-    "webpack-hot-middleware": "2.21.0"
+    "webpack-hot-middleware": "2.22.2"
```
Comment (samouss) [inline community-website/src/community-project-boilerplate-docgen/package.json]: We should pin all the dependencies for this package.

File: community-website/package.json [hunk 3]
```diff
@@ -12,17 +12,16 @@
     "test": "yarn lint && yarn docs:build && git checkout docs && git clean -f docs"
   },
   "devDependencies": {
-    "algolia-aerial": "^1.2.12",
-    "babel-eslint": "8.0.3",
-    "doctoc": "1.3.0",
-    "eslint": "4.12.1",
-    "eslint-config-algolia": "13.0.1",
+    "algolia-aerial": "^1.3.4",
```
Comment (samouss) [inline community-website/package.json]: We should also pin this dependencies.

Comment (vvo) [reply community-website/package.json]: They will all be pinned by renovate (I added it yesterday)

File: .circleci/config.yml (hunk 1 above)
Comment (vvo) [reply .circleci/config.yml]: Trying to remove it, otherwise Ill add a comment

**Pull request merged**